adding new server adapter

anna-y-perion opened the pull request.

anna-y-perion committed:
adding new server adapter

saar120 commented on static/bidder-info/programmaticxortb.yaml:
```diff
@@ -0,0 +1,20 @@
+      - video
+userSync:
+  iframe:
+    url: https://sync.programmaticx.ai/api/user/html/6819bdc3e6bb44545c55f843?pbs=true&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}
```
Make sure the ID is the correct one

anna-y-perion committed:
fixing sync id

anna-y-perion committed:
adapter change

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
adapter change

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
New Adapter: adding alias adapter

anna-y-perion committed:
inheriting openrtb from parent

anna-y-perion committed:
related to the PR #4428
https://github.com/prebid/prebid-server/pull/4428
we had to change the name after existing PrebidJs adapter was refactored.

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
removing irrelevant file

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
Merge branch 'master' into adding_programmaticx_rtb_adapter

anna-y-perion committed:
fixing problems after https://github.com/prebid/prebid-server/pull/4352 merge
